Kentico.Kontent.Management/IManagementClient.cs

```diff
@@ -536,8 +544,9 @@ public interface IManagementClient
         /// <typeparam name="T">Type of the content item elements</typeparam>
         /// <param name="identifier">The identifier of the language variant.</param>
         /// <param name="variantElements">Represents inserted or updated strongly typed language variant elements.</param>
+        /// <param name="workflow">Workflow step definition to set the inserted or updated language variant.</param>
         /// <returns>The <see cref="LanguageVariantModel{T}"/> instance that represents inserted or updated language variant.</returns>
-        Task<LanguageVariantModel<T>> UpsertLanguageVariantAsync<T>(LanguageVariantIdentifier identifier, T variantElements) where T : new();
+        Task<LanguageVariantModel<T>> UpsertLanguageVariantAsync<T>(LanguageVariantIdentifier identifier, T variantElements, WorkflowStepIdentifier workflow) where T : new();
```

Comment:
`workflow` can be an optional parameter with default value `null`

Reply:
fixed